builtin/settingtypes.txt

```diff
@@ -812,7 +812,7 @@ fixed_map_seed (Fixed map seed) string
 #    Creating a world in the main menu will override this.
 #    Current mapgens in a highly unstable state:
 #    -    The optional floatlands of v7 (disabled by default).
```

Comment:
should trailgen be marked as unstable here?

Reply:
Yes, I think it would be good to mark it as unstable for now until more people play with it.